cohttp/src/response.ml

```diff
@@ -24,8 +24,12 @@ type t = {
   flush: bool;
 } [@@deriving fields, sexp]
 
-let make ?(version=`HTTP_1_1) ?(status=`OK) ?(flush=false) ?(encoding=Transfer.Chunked) ?headers () =
-  let headers = match headers with None -> Header.init () |Some h -> h in
+let make ?(version=`HTTP_1_1) ?(status=`OK) ?(flush=false) ?(encoding=Transfer.Chunked) ?(headers=Header.init ()) () =
+  let encoding =
+    match Header.get_transfer_encoding headers with
+    | Transfer.(Chunked | Fixed _)  as enc -> enc
+    | Unknown -> encoding
```

Comment:
Thanks, this looks like a strict improvement to me! Would it be possible to make it clear in the ocamldoc header for Response that the `headers` field has priority over the encoding field?  This would also be good to note in the CHANGES entry for this so that developers are aware of the change in case they happened to depend on the old behaviour.

The new one is obviously better though.

Reply:
Hi ! Thanks for the review. Just a quick question before pushing: where should I write in the CHANGES entry ?